### ISSUE TITLE
Require psycopg2 as lambda layer, remove psycopg2-binary

### ISSUE THREAD

**Issue by jackdbd** [jackdbd/hokuto-no-ken-api#26]
Turns out that [someone](https://github.com/jetbridge/psycopg2-lambda-layer) has already built a lambda layer for psycopg2, so it's better to close #16 and address this issue instead.

**Comment by jackdbd** [comment]
Closed by PR #27

**Comment by jackdbd** [comment]
For future reference, use this ARN:

```sh
arn:aws:lambda:eu-central-1:898466741470:layer:psycopg2-py37:6
```

from the [psycopg2-lambda-layer](https://github.com/jetbridge/psycopg2-lambda-layer) repo.
